src/renderer/qssWidget/js/qssCustomStepperBrightnessWidget.js

```diff
@@ -0,0 +1,342 @@
+                closeOnBlur: true
+            });
+        }
+    };
+
+    /**
+     * Invoked whenever the increment button is activated. Takes care of
+     * increasing the setting's value with the amount specified in the setting's
+     * schema, animating the button appropriately and/or firing an event if
+     * an attempt is made to increase value above the maximum allowed value.
+     * @param {Component} that - The `gpii.qssWidget.stepperBrightness` instance.
+     * @param {jQuery} button - The jQuery object repesenting the increment button
+     * in the QSS stepper widget.
+     */
+    gpii.qssWidget.stepperBrightness.activateIncButton = function (that, button) {
+        var boundReached = that.increment();
+        that.animateButton(button, boundReached);
+        if (boundReached) {
+            // register bound hit
+            that.boundReachedHits += 1;
+
+            that.events.onUpperBoundReached.fire();
+        } else {
+            that.boundReachedHits = 0;
+        }
+    };
+
+    /**
+     * Invoked whenever the decrement button is activated. Takes care of
+     * decreasing the setting's value with the amount specified in the setting's
+     * schema, animating the button appropriately and/or firing an event if
+     * an attempt is made to decrease value below the minimum allowed value.
+     * @param {Component} that - The `gpii.qssWidget.stepperBrightness` instance.
+     * @param {jQuery} button - The jQuery object repesenting the decrement button
+     * in the QSS stepper widget.
+     */
+    gpii.qssWidget.stepperBrightness.activateDecButton = function (that, button) {
+        var boundReached = that.decrement();
+        that.animateButton(button, boundReached);
+        if (boundReached) {
+            // register bound hit
+            that.boundReachedHits += 1;
+
+            that.events.onLowerBoundReached.fire();
+        } else {
+            that.boundReachedHits = 0;
+        }
+    };
+
+    /**
+     * Either increases or decreases the current setting's value (depending on the
+     * `shouldSubtract` parameter) with the `divisibleBy` amount specified in the
+     * setting's schema. It also takes care that the new value of the setting does
+     * not become bigger/smaller than the maximum/minimum allowed value for the
+     * setting.
+     * @param {Component} that - The `gpii.qssWidget.stepperBrightness` instance.
+     * @param {Number} value - The initial value of the setting before the operation.
+     * @param {Object} schema - Describes the schema of the setting.
+     * @param {Number} schema.min - The minimum possible value for the setting.
+     * @param {Number} schema.max - The maximum possible value for the setting.
+     * @param {Number} schema.divisibleBy - The amount which is added or subtracted
+     * from the setting's value every time this function is invoked.
+     * @param {Boolean} shouldSubtract - Whether the `divisibleBy` amount should be
```

Comment:
Slightly cleaner both in the interface and implementation here is to ask for either +1 or -1 as a multiplier rather than a boolean

Reply:
Done